org/all-prs.ts

```diff
@@ -1,9 +1,25 @@
-import { schedule } from "danger"
+import { schedule, danger, warn, fail } from "danger"
 
-// Check for dependency changes
+const pr = danger.github.pr
+const bodyAndTitle = (pr.body + pr.title).toLowerCase()
+
+// Custom modifiers for people submitting PRs to be able to say "skip this"
+const wipPR = bodyAndTitle.includes("wip ") || bodyAndTitle.includes("[wip]")
+
+// Highlight package dependencies on Node projects.
 import yarn from "danger-plugin-yarn"
 schedule(yarn())
 
-// Check out docs
+// Keep our Markdown documents awesome
 import spellcheck from "danger-plugin-spellcheck"
 schedule(spellcheck({ settings: "artsy/artsy-danger@spellcheck.json" }))
+
+//  RFC: #5 - No PR is too small to warrant a paragraph or two of summary
```

Comment:
This is cool - sorta helps someone follow the papertrail. What would you think about including a link to the issue for the RFC? That way someone could go right back to the discussion that spawned the rule.